specification/sigma-appendix-modifiers.md

```diff
@@ -0,0 +1,148 @@
+# Sigma Modifiers
+
+The following document defines the standardized modifiers that can be used in Sigma.
+
+- Version 2.1.0
+- Release date 2025-08-02
+
+<!-- mdformat-toc start --slug=github --no-anchors --maxlevel=6 --minlevel=2 -->
+
+- [Generic Modifiers](#generic-modifiers)
+- [String Modifiers](#string-modifiers)
+  - [Regular Expression](#regular-expression)
+  - [Encoding](#encoding)
+- [Numeric Modifiers](#numeric-modifiers)
+- [Time Modifiers](#time-modifiers)
+- [IP (Internet Protocol) Modifiers](#ip-internet-protocol-modifiers)
+- [Specific Modifiers](#specific-modifiers)
+- [History](#history)
+
+<!-- mdformat-toc end -->
+
+### Generic Modifiers
+
+The following modifiers are considered generic modifiers and can be applied on all types of fields.
+
+- `all`: Normally, lists of values are linked with *OR* in the generated query. This modifier
+  changes this to *AND*. This is useful if you want to express a command line invocation with different
+  parameters where the order may vary and removes the need for some cumbersome workarounds.
+
+  Single item values are not allowed to have an `all` modifier as some back-ends cannot support it.
+  If you use it as a workaround to duplicate a field in a selection, use a new selection instead.
+
+- `startswith`: The value is expected at the beginning of the field's content. (replaces e.g. 'adm\*')
+
+- `endswith`: The value is expected at the end of the field's content (replaces e.g. '\*\\cmd.exe')
+
+- `contains`: Puts `*` wildcards around the values, such that the value is matched anywhere in the
+  field.
+
+* `exists`: Defines that a certain field has to exist or must not exist in a log event by providing a boolean value. Note that this check only verifies the presence of a field, not its value, be it empty or null.
+* `cased`: Values are applied case sensitively. Default Sigma behavior is case-insensitive matching.
+* `neq`: The field is different from the value. It's can be used with string or number.
```

Comment:
"from the specified values" - generally Sigma allows lists, no differences here.